src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerTeamController.cs

```diff
@@ -352,5 +358,21 @@ public ActionResult CreateVacancy(AccountDashboardViewModel model)
         {
             return PartialView(model);
         }
+        [HttpGet]
+        [Route("EmulatedReserveFunding")]
```

Comment:
Please move these methods to the EmulatedFundingJourneyController and update the link to point to them.

Reply:
Deleted the EmulatedReserveFunding method the others need to be here to allow for receiving of data